ifrs17/Utils/Extensions.ipynb

```diff
@@ -75,8 +75,7 @@
         {
             "cell_type": "code",
             "source": [
-                "// because the default(T) is something else than the first/last element. What about \"static T GetValidElement<T>(this IList<T> array, int index)\"? ",
-                "\nstatic T GetElementOrDefault<T>(this ICollection<T> array, int index)",
+                "static T GetValidElement<T>(this IList<T> array, int index)",
```

Comment:
why the change in type? from ICollection to IList?

Reply:
I reverted this change